Automattic/wp-e2e-tests
# Add back circleCIToken to encrypted config

circleCIToken was accidentally dropped from the config file. I added one back in.

To Test:

Get a CONFIG_KEY from SS via searching for E2E Config decode key
decode a config with `npm run decryptconfig`
Ensure output contains `circleCIToken`
